scripts/windows_installer.py

```diff
@@ -0,0 +1,104 @@
+            self.THYRA_APP_URL = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_windows-amd64.exe"
+            self.MKCERT_URL = "https://dl.filippo.io/mkcert/latest?for=windows/amd64"
+        else:
+            self.printErrorAndExit(f"Unsupported architecture {platform.machine()}")
+
+    def printErrorAndExit(self, error):
+        logging.error(error)
+        os.system("pause")
+        os._exit(-1)
+
+    def unzipAcrylic(self):
+        logging.info("Unzipping Acrylic...")
+        try:
+            with zipfile.ZipFile(self.ACRYLIC_DNS_PROXY_FILENAME, 'r') as zip_ref:
+                zip_ref.extractall(self.DEFAULT_ACRYLIC_PATH)
+        except:
+            self.printErrorAndExit("Could not unzip Acrylic")
+
+    def configureAcrylic(self):
+        logging.info("Configuring Acrylic...")
+        f = open(os.path.join(self.DEFAULT_ACRYLIC_PATH, self.ACRYLIC_HOST_FILE), "r+")
+        if f.read().find("127.0.0.1 *.massa") != -1:
+            f.close()
+            return
+        f.write("\n127.0.0.1 *.massa")
+        f.close()
+
+        logging.info("Restarting Acrylic service...")
+        self.executeCommand("NET STOP AcrylicDNSProxySvc & NET START AcrylicDNSProxySvc", True)
+
+    def configureNetworkInterface(self):
+        logging.info("Configuring network interface...")
+        commandOutput = self.executeCommand("wmic nic where \"netenabled=true\" get netconnectionID", True)
+
+        networkAdpatersNames = list(filter(None, commandOutput.split('\n')))
+        networkAdpatersNames.pop(0)
+        networkAdpatersNames = [name.strip() for name in networkAdpatersNames]
+
+        for name in networkAdpatersNames:
+            self.executeCommand(f'NETSH interface ipv4 set dnsservers "{name}" static 127.0.0.1 primary')
+            self.executeCommand(f'NETSH interface ipv6 set dnsservers "{name}" static ::1 primary')
+
+        logging.info("Network interface configured")
+
+    def setupDNS(self):
+        logging.info("Setting up DNS...")
+
+        if os.path.exists(self.DEFAULT_ACRYLIC_PATH):
+            logging.info("Acrylic DNS Proxy is already installed")
+        else:
+            cwd = os.getcwd()
+
+            logging.info("Installing Acrylic DNS Proxy...")
+            self.downloadFile(self.ACRYLIC_DNS_PROXY_URL, self.ACRYLIC_DNS_PROXY_FILENAME)
+            self.unzipAcrylic()
+
+            os.chdir(self.DEFAULT_ACRYLIC_PATH)
+            self.executeCommand(os.path.join(self.DEFAULT_ACRYLIC_PATH, "InstallAcrylicService.bat"))
+            self.configureNetworkInterface()
+            os.chdir(cwd)
+
+            try:
+                os.remove(self.ACRYLIC_DNS_PROXY_FILENAME)
+            except:
+                logging.error("Could not remove Acrylic DNS Proxy installer")
+
+        self.configureAcrylic()
+
+
+if __name__ == "__main__":
+    if platform.system() != "Windows":
+        logging.error("This script is only compatible with Windows")
```

Comment:
Why don't you use the helper function printErrorAndExit() ?

Reply:
For now, the `printErrorAndExit` is in the Installer class, that's why I'm not using it there. 
I will find a clean way to call it while making sure it's still compatible with every platforms. (I will probably just make it as a static method)